shared/services/config/nethermind-params.go

```diff
@@ -128,6 +131,17 @@ func NewNethermindConfig(cfg *RocketPoolConfig) *NethermindConfig {
 			OverwriteOnUpgrade: false,
 		},
 
+		FullPruningMaxDegreeOfParallelism: config.Parameter{
+			ID:                 "fullPruningMaxDegreeOfParallelism",
+			Name:               "Full pruning parallelism",
+			Description:        "The number of threads allocated to concurrently by Nethermind to prune data.",
+			Type:               config.ParameterType_Uint,
```

Comment:
-1 is a valid option 
![Sem título](https://github.com/user-attachments/assets/b75713e4-c0ef-4345-9da5-9d48c5564b96)